src/QuickMarcEditor/utils.test.js

```diff
@@ -1113,4 +1167,14 @@ describe('QuickMarcEditor utils', () => {
       expect(utils.getContentSubfieldValue('$a Test Title')).toEqual({ $a: 'Test Title' });
     });
   });
+
+  describe('validateLocationSubfield', () => {
+    it('should return true for valid location subfield', () => {
+      expect(utils.validateLocationSubfield({ content: '$b VA/LI/D ' }, locations));
+    });
+
+    it('should return false for locations that do not exist', () => {
+      expect(utils.validateLocationSubfield({ content: '$b NOT/VA/LI/D ' }, locations));
+    });
+  });
```

Comment:
Shouldn't `expect`s in this `describe` block have some assertive functions, e.g. `.toBe(true)`? Or am I missing something?

Reply:
@mariiaschv you're right, don't know how I missed it